src/default-config.json

```diff
@@ -1,9 +1,7 @@
 {
   "dataOption": "live",
-  "staticFilePath": "enter the pathway to static binary file with BGP data",
-  "staticFilesLink": "http://routeviews.org/route-views.ny/bgpdata/2021.11/UPDATES/",
-  "outlierDetectionAlgorithm": "mad",
-  "windowSize": "30",
-  "subscriptions": [{"prefix": "151.101.0.0/16"}, 
-                    {"path": 8137}]
+  "madParameters": 10, 
+  "maxBuckets": 20,
+  "windowSize": 360,
+  "subscriptions": [{"prefix": "0.0.0.0/0"}]
```

Comment:
suggest setting this to an ASN filter so that there's an example of it: `{"path": "54113$"}`